draft-schedulers.mkd

```diff
@@ -305,19 +321,19 @@ Multipath TCP {{ACMCS14}} indicate that it does not provide good performance.
 This packet scheduler uses one additional state at the connection level:
 last_path. This stores the identifier of the last path that was used to send a
 packet. We assume that the paths are identified by an integer. The scheduler is
-defined by the pseudocode shown in {{fig-rr}}.
+defined by the code shown in {{fig-rr}}.
 
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Packet Arrival:  # data packet of length l
-begin:
-  last_path++  # select next path
-  if(not_blocked(path[last_path])):
-        send packet over last_path
-  else:
-        goto begin: # try next path
-
+class RoundRobin(Scheduler):
+    """ Chooses an available path in a round-robin manner between two paths. """
```

Comment:
I just re-read the introduction saying that it is there on purpose. When I read the code, I was proposing to adapt this because this scheduler is the only one not supporting multiple subflows.